content/pages/resources.md

```diff
@@ -73,3 +87,5 @@ The [Cleveland PyLadies YouTube channel](https://www.youtube.com/channel/UCrX6AA
 2018_09_06: Raise Your Voice - [Mini-Workshop on Being Visible in Tech](http://meetu.ps/e/FLvfB/DxhG9/f)  
    [Write your first talk proposal](https://docs.google.com/presentation/d/19BdDbowE1Ec9bbDyD9LfhYsSUllm5qera-eH2APkk28/edit?usp=sharing)  
    [Why/How to Keep a Tech Blog](https://docs.google.com/presentation/d/1KPfsylMuSTvbIzJULoZCJS4rTpbm6M2rauXJAvdwrjk/edit?usp=sharing)
+
+[Medium](https://medium.com/) offers a simple platform for publishing blog posts.  Check out their [best practices for writing](https://blog.medium.com/best-practices-for-writing-on-medium-386506ae62b9) along with their [help articles on formatting and publishing](https://help.medium.com/hc/en-us/categories/200058025-Writing).
```

Comment:
This section currently has a header indicating this resource is a slide from a previous event. We should create a heading for tech writing, “other,” or something else and either move this resource there or remove the current heading and put all of these in that catch-all.